### DISCUSSION TITLE
Trouble with sending screenshot to Imgur

### DISCUSSION THREAD

**Discussion by nvkurganova** [flameshot-org/flameshot#2412] [Q&A] [answered]
I have downloaded zip version for Windows, unpacked it.
When I click on icon send to Imgur, I see an error "Error transferring https://api.imgur.com/3/image?title=&description=2022-02-11_16-09 - server replied: nginx". What it can be?

**Comment by mmahmoudian** [comment, marked as answer]
It would have been useful to know where did you get that error (i.e in terminal or in notification). Also this is most probably due to Imgur and has nothing to do with us. Do you will have the issue?

**Update:** I added a documentation page to our website to illustrate how to handle this situation:
https://flameshot.org/docs/guide/imgur-help/

**Reply by mrosenstihl** [reply]
Hi, I have the same error on Debian Bullseye (Flameshot version 0.9.0). The error comes as a pop up, there is no info on the command line.
This seems to be related to #188 .

**Reply by mmahmoudian** [reply]
Please read: https://flameshot.org/docs/guide/imgur-help/

**Comment by nvkurganova** [comment]
HelloNow it works 🤷‍♀️ Thanks for answer 16.02.2022, 22:43, "Mehrad Mahmoudian" ***@***.***>: It would have been useful to know where did you get that error (i.e in terminal or in notification). Also this is most probably due to Imgur and has nothing to do with us. Do you will have the issue?—Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.You are receiving this because you authored the thread.Message ID: ***@***.***>
